api/_bot/middleware/reply-to.js

```diff
@@ -1,5 +1,5 @@
 export default (ctx, next) => {
-  ctx.replyMessage = (msg, opts = {}) =>
```

Comment:
this does not reply to a user in a sense of `@user <message>` replies to the message itself so it does not make sense to rename it

Reply:
reverted